spec/tapioca/compilers/dsl/active_record_associations_spec.rb

```diff
@@ -407,7 +407,42 @@ def commenters_attributes=(attributes); end
     end
 
     describe("with errors") do
-      it("generates RBI file for broken has_many :through collection association with errors") do
+      it("generates RBI file for broken has_one association") do
+        add_ruby_file("schema.rb", <<~RUBY)
+          ActiveRecord::Migration.suppress_messages do
+            ActiveRecord::Schema.define do
+              create_table :posts do |t|
+                t.references(:author)
+              end
+            end
+          end
+        RUBY
+
+        add_ruby_file("post.rb", <<~RUBY)
+          class Post < ActiveRecord::Base
+            has_one :author
```

Comment:
Can we also add the cases for `has_many` and `belongs_to` and `has_and_belongs_to_many`?
```suggestion
            has_one :author
            has_many :comments
            belongs_to :blog
            has_and_belongs_to_many :commenters
```